ui/ui-components/pages/setup.tsx

```diff
@@ -37,10 +35,16 @@ export default function Page(props) {
   );
 
   async function load() {
+    const {
+      models: [model],
+    } = await execApi<Actions.ModelsList>("get", "/models", {
+      limit: 1,
+    });
+
     const response = await execApi(
       "get",
       `/setupSteps`,
-      { modelId: navigationModel.value },
+      { modelId: model?.id },
```

Comment:
I don't think it's necessary to have this as part of the action input anymore since the model won't be coming from the UI. It can probably be determined somewhere in the action or closer to where it's being used.